app/components/layout/SideTopBanner.tsx

```diff
@@ -10,6 +10,7 @@ export type ISideTopBanner = {
   headerInnhold?: ReactNode;
   knappIBanner?: ReactNode;
   tilbakeKnapp?: boolean;
+  tittelElementer?: ReactNode[];
```

Comment:
Er det greit å legge til denne for å få elementer rett etter tittelen? Som flytter seg etter lengden på tittel?